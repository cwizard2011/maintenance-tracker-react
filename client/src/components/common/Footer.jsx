import React from 'react';

const Footer = () => (
  <footer>
    <div>
        Maintenance Tracker by Adeoye Peter
    </div>
  </footer>
);

export default Footer;
