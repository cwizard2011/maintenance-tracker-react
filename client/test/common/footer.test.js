import React from 'react';
import { shallow } from 'enzyme';
import Footer from '../../src/components/common/Footer';

describe('Guest Navigation', () => {
  it('render guest navigation correctly', () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper).toMatchSnapshot();
  });
});
