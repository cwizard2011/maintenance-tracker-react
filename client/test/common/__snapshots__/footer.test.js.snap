// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Guest Navigation render guest navigation correctly 1`] = `
<footer>
  <div>
    Maintenance Tracker by Adeoye Peter
  </div>
</footer>
`;
